studio/components/interfaces/TableGridEditor/SidePanelEditor/ForeignKeySelector/ForeignKeySelector.tsx

```diff
@@ -240,6 +272,25 @@ const ForeignKeySelector: FC<Props> = ({ column, visible = false, closePanel, sa
                   ))}
                 </Listbox>
               )}
+              <SidePanel.Separator />
+              <Listbox
+                id="deletionAction"
+                value={selectedForeignKey.deletionAction}
+                label="Deletion action if referenced row is removed"
```

Comment:
I think this should be "Delete action if...."

Reply:
I realised "Delete action if referenced row is removed" kinda sounds weird as well now that I think twice about it
- kinda implicitly says that my data might get deleted if the reference row is removed, but the selection here is more of a "response" if the reference row is removed

Reply:
Maybe just "Action if reference row is removed for now"